src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -182,6 +182,8 @@ import CommonHead from '../components/CommonHead.astro';
 </html>
 ```
 
+Note that explicitly setting the [transition:name](#naming-a-transition) is necessary here, as the `<html>` element is directly used within in an individual page, not in a shared layout component.
```

Comment:
So my thinking is, instead of explaining why naming is necessary here, the "naming a transition" section should specify when / why names are necessary!

It currently says:

![image](https://github.com/user-attachments/assets/802cb9f9-cc2a-4f6d-872e-dd52477fc9ed)

The reason you mentioned above doesn't seem to be addressed in that. I would say, no explanation here. Just show the example with the `name="root"`. BUT maybe we need to add something to when names are necessary to explain this case? (If that is done, then I think this explanation stands on its own. Even if people see this and are like, "Why the name?" then they can check the docs for naming a transition and the explanation should make sense there.)

Reply:
A great, thank you very much!

That is a very good point! People can look up `transition:name` themselves when they are irritated here! 

Aside: The order of `name` and `animate` in the `<html>` element is arbitrary. I have no strong feeling which one is more natural for a reader looking for `transition:animate="none"`. I put `transition:animate="none"` at the end to make it stand out more clearly. But would be happy to re-order if that would be clearer.  

For improving the _Naming a transition_ section:
I never understood when automatically generated names would match. I recommend to always explicitly set names using `transition:name`. But that is probably not what we want to write in _Naming a transition_ ;-)